sdk/python/feast/feature_store.py

```diff
@@ -1179,14 +1180,19 @@ def get_online_features(
             for k, v in join_key_python_values.items()
         }
 
-        # Populate result rows with join keys
-        result_rows = [
-            GetOnlineFeaturesResponse.FieldValues() for _ in range(len(entity_rows))
-        ]
+        # Populate online features response proto with join keys
+        online_features_response = GetOnlineFeaturesResponse(
+            results=[
+                GetOnlineFeaturesResponse.FeatureVector()
+                for _ in range(len(entity_rows))
+            ]
+        )
         for key, values in join_key_proto_values.items():
-            for row_idx, result_row in enumerate(result_rows):
-                result_row.fields[key].CopyFrom(values[row_idx])
-                result_row.statuses[key] = FieldStatus.PRESENT
+            online_features_response.metadata.feature_names.val.append(key)
+            for row_idx, result_row in enumerate(online_features_response.results):
+                result_row.values.append(values[row_idx])
+                result_row.statuses.append(FieldStatus.PRESENT)
+                result_row.event_timestamps.append(Timestamp())
```

Comment:
So python feature server won't provide real event timestamp? and correct status as well

Reply:
ah, this is entities. I got it.
But why we put entities in feature vectors?

Reply:
Looks like this note got lost during the conflict resolution:
```
# Note: We temporarily store entities in the same field as features for backwards compatibility.
# Java server uses the same GetOnlineFeaturesResponse but does not return entities right now.
# We need to unify the logic between Python & Java on this.```
Basically Python SDK used to return entities as part of the online features response and I don't want to break that with this PR. Although we should have a followup conversation about how to properly resolve this.